wls-basisdaten-service/src/test/java/de/muenchen/oss/wahllokalsystem/basisdatenservice/utils/Authorities.java

```diff
@@ -79,10 +85,17 @@ public class Authorities {
             REPOSITORY_DELETE_WAHLTAG
     };
 
+    public static final String[] ALL_AUTHORITIES_RESET_WAHLEN = new String[] {
```

Comment:
die Variante ohne ohne `new String[]` finde ich schöner. Kannst du aber auch drin lassen da es bereits gemischt ist.

Reply:
Vereinheitlicht 310f6d1351f2c0e0053858b3912775175d1a6b2d

Reply:
lob für den Eifer und du kannst es so lassen, aber bitte in Zukunft daran denken keine Issue-Fremden stellen anzupassen. Das in einen kleinen PR separat machen.